main.go

```diff
@@ -218,17 +221,18 @@ func main() {
 		Certificates: dcrwCerts,
 		DisableTLS:   false,
 	}
-	dcrwClient, err = rpcclient.New(connCfgDaemon, nil)
+	rpcClient, err := rpcclient.New(connCfgDaemon, nil)
 	if err != nil {
 		log.Errorf("Failed to start dcrd rpcclient: %s", err.Error())
```

Comment:
This comment is incorrect and should refer to dcrwallet, not dcrd. Unrelated to this PR, but we might as well fix while you are making changes here.

There are two more instances above this one.